src/lightning/pytorch/strategies/ddp.py

```diff
@@ -378,6 +378,14 @@ def register_strategies(cls, strategy_registry: _StrategyRegistry) -> None:
                 start_method=start_method,
             )
 
+        strategy_registry.register(
```

Comment:
That's already done in the code lines above. Did you mean to add this to `src/lightning/fabric/strategies/ddp.py` maybe?

Reply:
> That's already done in the code lines above. Did you mean to add this to `src/lightning/fabric/strategies/ddp.py` maybe?

Oh, I am sorry. i made the changes in the wrong file.